docs/csharp/language-reference/index.md

```diff
@@ -55,7 +55,10 @@ The features that have been implemented in C# versions after 6.0 are represented
   
  [C# 8.0 Specification Proposals](../../../_csharplang/proposals/csharp-8.0/nullable-reference-types.md)  
  C# 8.0 is available with .NET Core 3.0. The features include nullable reference types, recursive pattern matching, default interface methods, async streams, ranges and indexes, pattern based using and using declarations, null coalescing assignment, and readonly instance members.
-  
+
+ [C# 9.0 Specification Proposals](../../../_csharplang/proposals/csharp-9.0/records.md)  
+ C# 9.0 is available with .NET 5.0. The features include records, top-level statements, nullable reference types, pattern matching enhancements, init only setters, target-typed new expressions, module initializers, extending partial methods, static anonymous functions, target-typed conditional expressions, covariant return types, extension GetEnumerator in foreach loops, lambda discard parameters, attributes on local functions, native sized integers, function pointers, suppress emitting localsinit flag, and unconstrained type parameter annotations.
```

Comment:
Nullable Reference types was in C# 8:

```suggestion
 C# 9.0 is available with .NET 5.0. The features include records, top-level statements, pattern matching enhancements, init only setters, target-typed new expressions, module initializers, extending partial methods, static anonymous functions, target-typed conditional expressions, covariant return types, extension GetEnumerator in foreach loops, lambda discard parameters, attributes on local functions, native sized integers, function pointers, suppress emitting localsinit flag, and unconstrained type parameter annotations.
```

Reply:
Nullable ref types spec is in the C# 9 TOC node -- should it not be there?
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/nullable-reference-types-specification

Reply:
Mads updated the spec, but the behavior is (mostly) the same. 

Sadly, this is a point in time problem will the ECMA committee catches up.